src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -152,5 +152,23 @@ public class ClassActions
             };
             return RenameClass;
         }
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetReplaceMethodModifiersAction(string methodName, string modifiers)
+        {
+            Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> ReplaceMethodModifiers = (SyntaxGenerator syntaxGenerator, ClassDeclarationSyntax node) =>
+            {
+                var allMembers = node.Members.ToList();
+                var allMethods = allMembers.OfType<MethodDeclarationSyntax>();
+                var replaceMethod = allMethods.Where(m => m.Identifier.ToString() == methodName).FirstOrDefault();
+                allMembers.Remove(replaceMethod);
+                SyntaxTokenList tokenList = new SyntaxTokenList(SyntaxFactory.ParseTokens(modifiers));
+                var newMethod = replaceMethod.WithModifiers(tokenList);
```

Comment:
What if the class or empty, or we didn't find the method in the class? Can replaceMethod be null?

Reply:
It could, this has been fixed as well after you mentioned the first 2 problems. Thanks.